geode-lucene/src/test/java/org/apache/geode/cache/lucene/LuceneIndexMaintenanceIntegrationTest.java

```diff
@@ -269,7 +271,7 @@ public void entriesFlushedToIndexAfterWaitForFlushCalled() throws InterruptedExc
     LuceneTestUtilities.resumeSender(cache);
     assertTrue(luceneService.waitUntilFlushed(INDEX_NAME, REGION_NAME, WAIT_FOR_FLUSH_TIME,
         TimeUnit.MILLISECONDS));
-    assertEquals(4, index.getIndexStats().getCommits());
+    assertEquals(8, index.getIndexStats().getCommits());
```

Comment:
We will need to figure out what to do with this stat... are commits() a useful stat? The additional 4 should probably be noted as being from the "complete" file